### DISCUSSION TITLE
Insert month or year from "Insert Date and Time" dialog box

### DISCUSSION THREAD

**Discussion by TwiggyWan** [zim-desktop-wiki/zim-desktop-wiki#2696] [Ideas]
Hello,

I like to use Journal plugin to make monthly recaps of stuff I've written.

For example in november 2024 when I create a page what I do is insert link > type 'Journal:2024:11' and then when I consult the month page I can use backlinks to see what i've done.

My idea is to enhance the 'Insert Date and Time" dialog box to make the month and year clickable : 

![image](https://github.com/user-attachments/assets/8877c8cd-4fac-48fd-ac11-ece6cc5e3393)

That would make the linking process easier.

Alternatively, if I have the option to do that through a page template, that would be good enough, but unfortunately I don't know how to do it and would appreciate help in this area.

Thanks

**Comment by jaap-karssenberg** [comment]
You can provide your own template codes in the config file of the insert
date and time dialog. If you use wiki markup for the links that should work

Op vr 8 nov. 2024 17:55 schreef TwiggyWan ***@***.***>:

**Reply by TwiggyWan** [reply]
Hi, 

i have two issues with your suggestion

1. it doesnt work : 
![image](https://github.com/user-attachments/assets/38e6695e-1026-415e-be1e-666a3035818e)
![image](https://github.com/user-attachments/assets/e5de4eb3-f596-40a1-9a9e-74dd150ae7ca)
as you can see, the link is formatted correctly on screen, but still links to the day I selected in the insert date and time dialog.

2. I had to edit $HOME/.config/zim/dates.list, but reading the manual : https://zim-wiki.org/manual/Help/Config_Files.html it says 'The file $XDG_DATA/zim/dates.list gives a list of strftime formats' , so I expected to find the file in one of the XDG_DATA paths, not in XDG_CONFIG
